src/services/refactors/inlineLocal.ts

```diff
@@ -0,0 +1,173 @@
+            });
+        }
+        return [refactorInfo];
+    }
+
+    function getLocalInfo(file: SourceFile, program: Program, startPosition: number): Info | undefined {
+        const token = getTokenAtPosition(file, startPosition);
+        const maybeDeclaration = token.parent;
+        const checker = program.getTypeChecker();
+        if (isLocalVariable(maybeDeclaration)) {
+            return createInfo(checker, maybeDeclaration);
+        }
+        if (isIdentifier(token)) {
+            const symbol = checker.getSymbolAtLocation(token);
+            if (!symbol) return undefined;
+            const declaration = symbol.valueDeclaration;
+            if (isLocalVariable(declaration)) {
+                return createInfo(checker, declaration, token);
+            }
+            return undefined;
+        }
+        return undefined;
+    }
+
+    function isLocalVariable(parent: Node): parent is VariableDeclaration {
+        return isVariableDeclaration(parent) && isVariableDeclarationInVariableStatement(parent);
+    }
+
+    function createInfo(checker: TypeChecker, declaration: VariableDeclaration, token?: Identifier): Info | undefined {
+        const name = declaration.name;
+        const usages = getReferencesInScope(ts.getEnclosingBlockScopeContainer(name), name, checker);
+        return canInline(declaration, usages) ? {
+            declaration,
+            usages,
+            selectedUsage: token ? token : undefined
+        } : undefined;
+    }
+
+    function canInline(declaration: VariableDeclaration, usages: ReadonlyArray<Identifier>): boolean {
+        let hasErrors = false;
+        if (!declaration.initializer) hasErrors = true;
+        if (containsProhibitedModifiers(declaration.parent.parent.modifiers)) hasErrors = true;
+        ts.forEach(usages, usage => {
+            if (isAssigned(usage)) hasErrors = true;
+        });
+        return !hasErrors;
+    }
+
+    function isAssigned(usage: Identifier): boolean {
+        type AssignExpr = AssignmentExpression<AssignmentOperatorToken>;
+        const assignment: AssignExpr = findAncestor(
+            usage,
+            ancestor => isAssignmentExpression(ancestor)) as AssignExpr;
+        return assignment && assignment.left === usage;
+    }
+
+    function containsProhibitedModifiers(modifiers?: NodeArray<Modifier>): boolean {
+        return !!modifiers && !!modifiers.find(mod => mod.kind === SyntaxKind.ExportKeyword);
+    }
+
+    function getEditsForAction(context: RefactorContext, actionName: string): RefactorEditInfo | undefined {
+        const { file, program, startPosition } = context;
+        const info = getLocalInfo(file, program, startPosition);
+        if (!info) return undefined;
+        const { declaration, usages, selectedUsage } = info;
+        const edits: FileTextChanges[] = [];
+        switch (actionName) {
+            case inlineAllActionName:
+                edits.push(...getInlineAllEdits(context, declaration, usages));

```

Comment:
an alternative would be:
```ts
return {edits: [...getInlineAllEdits(context, declaration, usages)] };
```

Reply:
but if you change the returnType of getInlineAllEdits to `FileTextChanges[]`
then you can use directly
```ts
return {edits: getInlineAllEdits(context, declaration, usages) };
```